pennylane/math/matrix_manipulation.py

```diff
@@ -102,10 +102,12 @@ def expand_matrix(mat, wires, wire_order=None, sparse_format="csr"):
 
     """
 
+    qudit_dim = int(qml.math.shape(mat)[-1] ** 1 / (len(wires)))
```

Comment:
Is this the correct order of operations? 🤔 

```
>>> 2 ** 1 / 4
0.5
>>> 2 ** (1/4)
1.189207115002721
```

Reply:
I believe you are indeed right.